### PR TITLE
Include help for base_imu and base_heading_axis options

### PR DESCRIPTION
**Pull request by aseth1**
### Brief summary of changes
- added Help information for extra `-Calibrate` options

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2476)

### CODE REVIEW

File: Applications/opensense/opensense.cpp [hunk 1]
```diff
@@ -276,13 +276,16 @@ void PrintUsage(const char *aProgName, ostream &aOStream)
     aOStream << "-PropertyInfo, -PI                      Print help information for properties in setup files.\n";
     aOStream << "-ReadX, -RX  directory settings.xml     Parse Xsens exported files from directory using settingsFile.xml.\n";
     aOStream << "-ReadA, -RA  datafile.csv settings.xml  Parse single csv file provided by APDM using specified settingsFile.xml.\n";
-    aOStream << "-Calibrate, -C modelPoseFile.osim calibrationOrientations.sto.\n";
+    aOStream << "-Calibrate, -C modelPoseFile.osim calibrationOrientations.sto. <base_imu> <base_heading_axis>\n";
```
Comment (jenhicks) [inline Applications/opensense/opensense.cpp]: This is minor, but in the setup file for IK, it is called base_imu_label ... should make consistent.

File: Applications/opensense/opensense.cpp [hunk 2]
```diff
@@ -276,13 +276,16 @@ void PrintUsage(const char *aProgName, ostream &aOStream)
...
     aOStream << "                                        Calibrate the modelPoseFile.osim model by registering\n";
     aOStream << "                                        IMU frames whose orientations in the sensor world frame are\n";
     aOStream << "                                        specified in calibrationOrientations.sto. and assuming \n";
     aOStream << "                                        the model's default pose is the calibration pose. The resultant\n";
     aOStream << "                                        model with IMU frames registered is written to file as\n";
-    aOStream << "                                        calibrated_modelPoseFile.osim\n";
+    aOStream << "                                        calibrated_modelPoseFile.osim. Optional arguments for identifying the\n";
+    aOStream << "                                        base IMU, e.g. 'pelvis imu' and its heading axis as 'x', 'y', or 'z',\n";
```
Comment (jenhicks) [inline Applications/opensense/opensense.cpp]: by heading do you mean forward or magnetic north?

File: Applications/opensense/opensense.cpp [hunk 3]
```diff
@@ -276,13 +276,16 @@ void PrintUsage(const char *aProgName, ostream &aOStream)
...
+    aOStream << "                                        to align all the IMU data so that base imu's heading axis is in the X\n";
+    aOStream << "                                        direction of OpenSim ground.\n";
```
Comment (jenhicks) [inline Applications/opensense/opensense.cpp]: What happens if you don't include the optional arguments?

Comment (aseth1) [reply Applications/opensense/opensense.cpp]: If base imu is not specified (or empty) then no heading correction is applied. If the base imu is specified but no axis is provide, 'z' is assumed. Should I add this info?

**Comment by aseth1** [comment]
Thanks @jenhicks and @jimmyDunne!

**Pull request merged**